AKS HCI update - Issue 583 & 552 (#585)
microsoft/azure_arc: docs/azure_arc_jumpstart/azure_arc_k8s/aks_stack_hci/aks_hci_powershell/_index.md

* Perform a clean installation of the AksHci PowerShell module. To install the AksHci PowerShell module remove any previous versions by running the below commands:

  ```powershell
  Install-Module -Name Az.Accounts -Repository PSGallery -RequiredVersion 2.2.4
  Install-Module -Name Az.Resources -Repository PSGallery -RequiredVersion 3.2.0
  Install-Module -Name AzureAD -Repository PSGallery -RequiredVersion 2.0.2.128
  Install-Module -Name AksHci -Repository PSGallery
  Import-Module Az.Accounts
  Import-Module Az.Resources
  Import-Module AzureAD
  Import-Module AksHci
  Exit
  ```
  

> * After this is done, close all PowerShell windows and verify the installation by tunning the following:

Reviewer: ```suggestion
* Once completed, close all PowerShell windows and verify the installation by tunning the following:
```
